lib/services/source-data-extractor.js

```diff
@@ -66,7 +66,7 @@ SELECT * WHERE
             ?s ?p ?o .
         }
     }
-    ${resources.map(resource => generateNestedResourceSubQuery(uri, resource)).join('')}
+    ${nested ? resources.map(resource => generateNestedResourceSubQuery(uri, resource)).join('') : ''}
```

Comment:
Do we need this extra parameter? Running it always won't break the query. Not running it, must be a good reason for it.

Reply:
We need for clearing previously extracted data. I'll give you an example : 

First extraction, we have the following extracted data
```
?form project:name "Name" ;
  contact:person <http://contact-person/1> .
<http://contact-person/1> first:name "Patrick" .
```
If we clean source data without specifying that we don't want it going down relationships, we will lose `<http://contact-person/1> first:name "Patrick" .` which might be used somewhere else.

Maybe I'm overthinking it now that I think of it, because we duplicate data when extracting, we don't reuse URIs from step 1 ...